src/quo2/components/drawers/documentation_drawers/style.cljs

```diff
@@ -1,9 +1,14 @@
-(ns quo2.components.drawers.documentation-drawers.style)
+(ns quo2.components.drawers.documentation-drawers.style
+  (:require [react-native.safe-area :as safe-area]))
+
+(def outer-container
+  ;; Margin negative to fill the bottom-sheet container where this component is used
+  {:margin-bottom (- (+ (safe-area/get-bottom) 8))})
 
 (def container
   {:align-items        :flex-start
    :padding-horizontal 20})
 
 (def content
   {:margin-top    8
-   :margin-bottom 16})
+   :margin-bottom (+ (safe-area/get-bottom) 8)})
```

Comment:
We have been adding a wrong margin-bottom, it was 16, but it should have been the bottom + 8:
![image](https://github.com/status-im/status-mobile/assets/90291778/c999c3c1-86c8-4510-88ef-40b20cec58f9)